src/jit/compiler.h

```diff
@@ -661,23 +701,23 @@ class  LclVarDsc
     bool                IsFloatRegType() const
                         {
                             return
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
                                 lvIsHfaRegArg ||
```

Comment:
As i mentioned above lvIsHfaRegArg is redundant. We could define a new routine as follows?

bool lvIsHfaRegArg()
{
   return lvIsHfa() && lvIsRegArg;
}

 and the same could be used here without any #ifdefs.